plugins/outputs/wavefront/wavefront.go

```diff
@@ -92,44 +98,55 @@ type MetricPoint struct {
 }
 
 func (w *Wavefront) Connect() error {
+
+	if len(w.StringToNumber) > 0 {
+		log.Print("W! [outputs.wavefront] The string_to_number option is deprecated; please use the enum processor instead")
+	}
+
+	if w.Url != "" {
+		log.Printf("D! [outputs.wavefront] connecting over http/https using Url: %s", w.Url)
+		sender, err := wavefront.NewDirectSender(&wavefront.DirectConfiguration{
+			Server:               w.Url,
+			Token:                w.Token,
+			FlushIntervalSeconds: 5,
```

Comment:
Telegraf considers returning from Write() without error to be successful delivery of metrics, but with the extra layer of buffering we don't actually know if the metric was sent.  There may be some reduction in durability because of this, might be worth considering a sender implementation that sends immediately without buffering?